### PR TITLE
Replace autotools build system with raw Makefile

### PR DESCRIPTION
**Pull request by AlexLanzano**
This is somewhat of a WIP. The Makefile itself works for building sed-tools. however I had to comment out the encryption functions in order to get a successful build. Not sure if I should leave this in the commit or not.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# sed-tools
```
Comment (mauricegale) [inline .gitignore]: I think this file should be added in on a  separate commit

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+install:
+	@echo "TODO"
+
+.PHONY: uninstall
+uninstall:
+	@echo "TODO"
+
+.PHONY: clean
+clean:
+	rm -rf $(DEPDIR) $(BOOTLOADER_OBJECTS) $(OBJECTS)
```
Comment (mauricegale) [inline Makefile]: Where is bootloader_objects defined? Do we need this right now?

File: .gitignore (hunk 1 above)
Comment (AlexLanzano) [reply .gitignore]: Agreed! Will separate.

File: Makefile (hunk 2 above)
Comment (AlexLanzano) [reply Makefile]: Whoops! That was an artifact from copying over my Makefile from another project of mine
